src/main/webapp/feed.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+  <title>Message Feed</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/user-page.css">
+
+  <script>
+    // Fetch messages and add them to the page.
+    function fetchMessages() {
+      const url = '/feed';
+      fetch(url).then((response) => { return response.json() })
+      .then((messages) => {
+        const messageContainer = document.getElementById('message-container');
+        messageContainer.innerHTML = (messages.length == 0) 
+        ? '<p>There are no posts yet.</p>' : messageContainer.innerHTML = '';
+
+        messages.forEach((message) => {
+          const messageDiv = buildMessageDiv(message);
+          messageContainer.appendChild(messageDiv);
+        });
+      })
+      .catch((error) => {
+        console.log(error.message)
+        });
```

Comment:
Closing brackets should be on the same column as the beginning of the line with the opening bracket:

```
.catch((error) => {
  ...
});
```